CMakeLists.txt

```diff
@@ -287,16 +229,14 @@ elseif("${CMAKE_SYSTEM_NAME}" STREQUAL "CYGWIN" OR "${CMAKE_SYSTEM_NAME}" STREQU
 
         message(INFO " Compiling for Windows (${CMAKE_SYSTEM_NAME}, MSYSTEM=$ENV{MSYSTEM})... ")
 else()
-        set(FOREIGN_OS True)
-        set(COMPILED_FOR_FOREIGN_OS True)
-        message(WARNING " Compiling for Unknown O/S... (${CMAKE_SYSTEM_NAME})")
+        message(FATAL_ERROR " Unknown O/S... (${CMAKE_SYSTEM_NAME})")
```

Comment:
Could we possibly change this to clearly indicate that the platform isn’t supported by Netdata instead of just saying the OS is unknown? Given how easy it is for us to give good, descriptive, error messages with CMake, I’d like to try and ensure we’re doing so.